src/elements/profiles/Profiles.wc.svelte

```diff
@@ -502,8 +502,8 @@
     pointer-events: all;
   }
 
-  .vertical-line {
-    border-left: 4px solid black;
-    height: 100%;
-  }
+  // .vertical-line {
+  //   border-left: 4px solid black;
+  //   height: 100%;
+  // }
```

Comment:
why are these lines commented?

Reply:
unused selector and it gives this warning

![image](https://user-images.githubusercontent.com/62248851/209553316-75e7072c-7661-484d-a581-881d653dc16a.png)